app/views/external_users/claims/basic_fees/_fields.html.haml

```diff
@@ -28,6 +28,6 @@
 
       = f.fields_for :basic_fees, basic_fees.extra_fees.sort_by(&:position) do |extra_fee_fields|
         - @basic_fee_count += 1
-        - if present(extra_fee_fields.object).should_be_displayed?
+        - if present(extra_fee_fields.object).should_be_displayed? && !@claim.discontinuance?
```

Comment:
Could refactor to remove conditional logic from view to presenter, but **not-a-blocker**.
e.g. 
```
def display_extra_fees?
  return false if claim.discontinuance?
  should_be_displayed? 
end
```

Reply:
👍